molmaker.py

```diff
@@ -145,7 +145,7 @@ def checkopts(self, args=sys.argv):
             try:
                 self.gmxlibdir = os.environ["GMXLIB"] 
             except KeyError:
-                self.gmxlibdir = re.sub("/bin/grompp", "/share/gromacs/top", self.grompp)
+                self.gmxlibdir = re.sub("/bin/gmx", "/share/gromacs/top", self.gmx)
```

Comment:
Hmmm... But what if the user is running GMX 4.6 and there's no `self.gmx`?
Should we instead use the following?
```python
self.gmxlibdir = re.sub("/bin/.*$", "/share/gromacs/top", self.grompp[0])
```